OpenRA.Mods.Cnc/FileFormats/BlowfishKeyProvider.cs

```diff
@@ -90,9 +90,8 @@ static void KeyToBigNum(uint[] n, byte[] key, uint len)
 
 		static uint LenBigNum(uint[] n, uint len)
 		{
-			uint i;
-			i = len - 1;
-			while ((i >= 0) && (n[i] == 0)) i--;
+			uint i = len - 1;
```

Comment:
##### Minor nit:
`var` should be used since `len` is already a `uint`, "Use `var` when declaring temporary variables inside methods." is part of the [coding standard](https://github.com/OpenRA/OpenRA/wiki/Coding-Standard), and the code is being refactored already. Otherwise, this code may never be refactored towards the coding standard.
```suggestion
			var i = len - 1;
```

Reply:
I understand that's the nit @atlimit8, any other issue?